src/client/sandbox/cookie.js

```diff
@@ -4,7 +4,7 @@ import UrlUtil from '../utils/url';
 import Settings from '../settings';
 import * as Cookie from '../utils/cookie';
 import { isCrossDomainWindows } from '../utils/dom';
-import { SET_COOKIE as SET_COOKIE_CMD } from '../../service-msg-cmd';
+import { setCookie as SET_COOKIE_CMD } from '../../service-msg-cmd';
```

Comment:
Better do it this way:
```js
import * as COMMAND from '../../service-msg-cmd';
```
and use it as:
```js
COMMAND.setCookie
```
It would be nice to rename `service-msg-cmd` to `commands` and move it to `session` for better semantics (this will partially solve #68)